docs/auth-acl/realm-export.json

```diff
@@ -60,6 +60,15 @@
         "containerId": "WIPPKeycloak",
         "attributes": {}
       },
+      {
+        "id": "75ce82cb-04a4-4061-a1e1-4fb83ee211c0",
```

Comment:
@MyleneSimon I want to add the `developer` role to our already existing keycloak instance without re-importing the realm json file. Is it necessary to add the id or is it an auto-generated id? From the keycloak admin console, I can only add the role name and its description.

Reply:
@mohamedOuladi you just need to specify name and description, check out the description of this PR, there is a section about how to update an existing Keycloak instance without re-importing the realm ("Adding the new role in Keycloak")